scrapy/itemchecker.py

```diff
@@ -0,0 +1,63 @@
+"""
+Filter for Scrapy Items to be used by Feed slots.
+"""
+from itemadapter import ItemAdapter
+
+
+class ItemChecker:
+    """
+    This will be used by FeedExporter to decide if an item should be allowed
+    to be exported to a particular feed.
+    :param feed_options: feed specific options passed from FeedExporter
+    :type feed_options: dict
+    """
+
+    item_classes = ()
+
+    def __init__(self, feed_options):
+        self.feed_options = feed_options
+
+        if 'item_classes' in self.feed_options:
+            self.item_classes += self.feed_options['item_classes']
```

Comment:
This will fail, since `self.item_classes` is a (non-mutable) tuple.

It may make sense to make `self.item_classes` a set instead of a tuple.

I also wonder if we should support item classes to be import paths instead of actual classes, in case the FEEDS setting has been defined as a JSON string.

Reply:
That's a good idea I'll include both the cases.

Reply:
My reasoning for using ```+=```: I was planning on having item_classes declared in a tuple in settings.py so I thought I would update the class's self.item_classes tuple with the operation ```+=```.